content/source/docs/enterprise/install/installer.html.md

```diff
@@ -226,7 +226,7 @@ ADD finalize_custom_worker.sh /usr/local/bin/finalize_custom_worker.sh
 Terraform Enterprise can store its state in a few different ways, and you'll
 need to decide which works best for your installation. Each option has a
 different approach to
-[recovering from failures](../system-overview/reliability-availability.html#recovery-from-failures-1).
+[recovering from failures](../system-overview/reliability-availability.html#recovery-from-failures).
```

Comment:
This link doesn't exist... maybe https://www.terraform.io/docs/enterprise/admin/automated-recovery.html?

```suggestion
[recovering from failures](../admin/automated-recovery.html).
```

Reply:
Sure, why not.